server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -291,8 +291,8 @@ public void putScanServerFileReferences(Collection<ScanServerRefTabletFile> scan
 
   @Override
   public Stream<ScanServerRefTabletFile> getScanServerFileReferences() {
-    try {
-      Scanner scanner = context.createScanner(DataLevel.USER.metaTable(), Authorizations.EMPTY);
+    try (
+        Scanner scanner = context.createScanner(DataLevel.USER.metaTable(), Authorizations.EMPTY)) {
```

Comment:
This closes the scanner, which causes an error `Scanner was closed` when the caller operates on the returned Stream.

Reply:
ooopppsss... seems I got a bit carried w/ applying the try catches w/o considering the context